packages/beta/src/api/alerts/alertsApi.ts

```diff
@@ -70,6 +76,10 @@ export class AlertsAPI extends BaseResourceAPI<Alert> {
     return this.channelsApi.list(filter);
   };
 
+  public sortAlerts = async (sort: AlertSortQuery) => {
+    return this.list(undefined, sort);
+  };
```

Comment:
what is the meaning of this one? sort without the filter, or?

Reply:
sort and list are normally the same endpoint, hence it should be a single method as well.
I pushed a change